packages/core/src/authorizer.ts

```diff
@@ -152,7 +152,7 @@ export function authorizer<
                   redirectURI: authorization.redirect_uri,
                   clientID: authorization.client_id,
                 },
-                60,
+                Date.now() / 1000 + 60,
```

Comment:
I think this is correct as before with `60` dynamo would not set the item for the code